src/calculateRentalCost.js

```diff
@@ -26,9 +26,23 @@
  *
  * @return {number}
  */
+const LONG_TERM = 7;
+const LONG_TERM_DISCOUNT = 50;
+const SHORT_TERM = 3;
+const SHORT_TERM_DISCOUNT = 20;
 
 function calculateRentalCost(days) {
-  // write code here
+  const BASE_PRICE = days * 40;
```

Comment:
so `BASE_PRICE` depends on `days` which is function argument, so it will change for sure - we typically use uppercased variable names for things that aren't changing